lib/flipper/adapters/redis.rb

```diff
@@ -125,14 +125,14 @@ def read_feature_keys
       # Private: Gets a hash of fields => values for the given feature.
       #
       # Returns a Hash of fields => values.
-      def doc_for(feature)
+      def doc_for(feature, _pipeline: @client)
         @client.hgetall(feature.key)
```

Comment:
Should this use `_pipeline`?

Reply:
🤦 , yes absolutely...

Reply:
Fixed. Also since I noticed the doc mark the method as private, I droped the leading `_`.